docs/src/appendix/cast/account/delete.md

```diff
@@ -0,0 +1,21 @@
+# `delete`
+Delete an account from `accounts-file` and optionally his associated Scarb profile.
```

Comment:
```suggestion
Delete an account from `accounts-file` and optionally its associated Scarb profile.
```

Reply:
I see a lot of @THenry14's suggestions are resolved but not applied, although they seem accurate. Is there any reason for it? 😅